proxies/live/apiproxy/policies/Quota.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<Quota name="Quota" type="calendar">
```

Comment:
You should consider changing to the new shared from ratelimiting approach https://nhsd-confluence.digital.nhs.uk/display/APM/Rate+limiting+shared+flow